Cargo.toml

```diff
@@ -6,22 +6,42 @@ edition = "2021"
 rust-version = "1.67.0"
 
 [workspace.dependencies]
-anyhow = { version = "1.0.66" }
-clap = { version = "4.0.1", features = ["derive"] }
-itertools = { version = "0.10.5" }
+anyhow = { version = "1.0.69" }
+bitflags = { version = "1.3.2" }
+chrono = { version = "0.4.23", default-features = false, features = ["clock"] }
+clap = { version = "4.1.8", features = ["derive"] }
+colored = { version = "2.0.0" }
```

Comment:
How'd you choose which crates to move into the workspace? (E.g., are all of these used in > 1 crate?)

Reply:
Yes, every crate move in the workspace is used by more than 1 crate.